regen-network/regen-web
# perf: 993 fix project details layout shift

## Description

Closes: regen-network/regen-registry#993

| Before (mobile)  | After(mobile)  | Before (desktop)  | After (desktop)  | 
|---|---|---|---|
| ![mobile before](https://user-images.githubusercontent.com/530644/178518991-ae0e0268-70cc-4df7-85de-263e2c6e16be.png)  | ![mobile after](https://user-images.githubusercontent.com/530644/178519023-8072d212-f47e-445d-9969-63207f9bbfa1.png)  | ![desktop before](https://user-images.githubusercontent.com/530644/178518915-907dd9c6-0fc4-44fe-8114-834cc220202d.png)  | ![desktop  after](https://user-images.githubusercontent.com/530644/178519068-cc363e0e-1f3a-4031-8b4f-8dcd0932ba8a.png)  |
| ![mobile metric before](https://user-images.githubusercontent.com/530644/178519127-be11d1de-3db5-459f-8e51-d4dc5990e0b5.png)  |  ![mobile metric after](https://user-images.githubusercontent.com/530644/178519151-2fe63046-0de0-48e0-8b08-e6a288296104.png) | ![desktop metric before](https://user-images.githubusercontent.com/530644/178519184-62b4ac4e-1879-4d2b-8092-7f1d850809ff.png)  | ![desktop metric after](https://user-images.githubusercontent.com/530644/178519213-6bf1fc93-591c-4499-bf1b-efa99a45c8cc.png)  |

Fix layout shifting on project page, mostly by setting default height wherever needed (images, map, top section).
Some heights have several values based on our different breakpoints to minimize CLS.
Also tried to handle dynamic height based on optional content (like top link section).



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1. Go to project details page (ie: https://deploy-preview-1050--regen-registry.netlify.app/projects/wilmot)
2. Check how the page is loading
3. (Optional) run performance insights from the chrome dev tools

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
